### PR TITLE
issue #81 fixed

### LINKED ISSUE #81: gameControllers: Convert module to typescript
# Description
Please read all the types being used in the codebase (mostly in `types.d.ts`) and convert this module to typescript.
This issue is `ONLY` intended for **first-time contributors**.

### PR DESCRIPTION
**Pull request by AB000001**
## Description
Fixes #81. Converted gameControllers.js to gameControllers.ts

## Motivation and Context
Issue #81 fixed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 node_modules
 frontend/node_modules
 .DS_Store
-.env
\ No newline at end of file
+.env
+*.json
```
Comment (kuv2707) [inline .gitignore:5]: What prompted this change?

**Review by kuv2707 — COMMENTED** [review]
Great work!
Please take a moment to read the contributing guidelines (in the file contributing.md) and format the PR description and commit message accordingly.

The PR description should have the clause `Fixes #issueno`.

Once that is done, this can be merged

**Comment by kuv2707** [comment]
@AB000001 Still on it?

File: .gitignore (hunk 1 above)
Comment (AB000001) [reply .gitignore:5]: I had changed it for my local purpose. We can ignore these entries from. Gitignore

**Comment by AB000001** [comment]
> Great work! Please take a moment to read the contributing guidelines (in the file contributing.md) and format the PR description and commit message accordingly.
> 
> The PR description should have the clause `Fixes #issueno`.
> 
> Once that is done, this can be merged

Modified the Pull request description to include issue #81. and modified the commit remark also

**Comment by kuv2707** [comment]
@AB000001 The commits don't seem to follow the guidelines. 
There should only be one commit with a properly formatted message. The contributing docs can help you with it. 
You need to "squash" all the commits into one.

Also I'd expect only one file change as per the issue description, but here there are two.

**Comment by kuv2707** [comment]
It seems like this issue has been addressed in a recent commit, since the file is now in typescript.
In that case, it would be useless to work further on it.

You can still use this PR to learn some git commands, and I'll approve it once it is in good shape, so that it is counted as a contribution. (Just be sure not to "rebase" this branch).

I'll unassign you from the issue so that you can take up others.
Thanks for all the work!

**Comment by kuv2707** [comment]
closing due to inactivity

**Pull request closed without merging**